### ISSUE TITLE
How could I add shortcuts of portable apps in the win10 type start menu?

### ISSUE THREAD

**Issue by freezingbz** [valinet/ExplorerPatcher#3991]
I found I couldn't added portable apps shortcuts to the win10 type start menu. I even tried copying the created .lnk files to C:\Users\Username\AppData\Roaming\Microsoft\Windows\Start Menu\Programs but they didn't show (probably due to they are not registered in the system?). Just wondering if there is way to realize it? Thanks!

**Comment by Amrsatrio** [comment]
Right click the exe in its folder and click Pin to Start. Don't mess around with the shortcuts yourself, the pinning system does it for you. Works for me.

**Comment by freezingbz** [comment]
Thank you for replying. Sorry I forgot to mention that before turning to the way I mentioned, I did try to right-click the exe files, but some just didn't show Pin to Start option (while some did). Have no idea why some exe files are so different, and how to pin them..

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.  And what language your windows is in.
Please post explorer patcher version.

**Comment by freezingbz** [comment]
> Please post windows version and build. You can find this by running winver. Please include if you are running x86-64 or Arm. And what language your windows is in. Please post explorer patcher version.

Attached please find those information.
![图片](https://github.com/user-attachments/assets/fed3e9fb-26f8-41b5-b923-d82ba5f5cfe9)
And my laptop has a x64 AMD CPU (Ryzen 7 8845H). The windows 11 is running in Chinese. And the explorer patcher version is 22621.4317.67.1.

By the way, one example that couldn't be pinned to start menu is Station (https://github.com/getstation/desktop-app).

Thanks!

**Comment by pyrates999** [comment]
So this is what you've done so far?

1. install chinese version of windows 11 23h2
2. install latest EP version listed here: https://github.com/valinet/ExplorerPatcher/releases
3. install winget: https://learn.microsoft.com/en-us/windows/package-manager/winget/
4. install this via winget: https://github.com/getstation/desktop-app
5. go to directory that getstation is installed to and right click on its exe and try to pin to start menu, but it doesn't work?

**Comment by freezingbz** [comment]
Things I did like below:
1. In a laptop installed with Chinese version win11 23h2, install UniGetUI(https://github.com/marticliment/UniGetUI/) via winget (winget comes with win11)
2. Install EP, station and other stuff I need via UniGetUI
3. Get some other portable apps I want
4. Customize setting of EP, and start to pin the shortcut I want by right-click way. Most of them works, left a few behind.

**Comment by pyrates999** [comment]
Please show the steps you did for the one that did not work in windows 11 chinese version 23h2.  After changing to the windows 10 start menu in EP properties application.  Include what directory you browsed to where you right clicked on the exe file and tried to pin it, and it did not show up in the windows 10 start menu.

**Comment by freezingbz** [comment]
Thank you for your patience. After some try, I think I figure it out. Just write down some notes here:
1. Right-click the exe file in the file manager (term 文件资源管理器 in Chinese version, don't know the exact name in English version), and click Pin to Start (if it has this option) works.
2. In some cases, it has no Pin to Start option. I guess it is caused by one of below: (1) There is already a shortcut in the start menu - all program column; (2) A lnk file exists in the wrong place that block this option to show.
3. For 2(1), I tried right-clicking the shortcut in the all program column then clicking pin. It worked.
4. For 2(2), what I did was searching out the wrong lnk file via everything and deleted it. Now step 1 could show the pin option.
